src/omni_epd/displays/inky_display.py

```diff
@@ -49,6 +51,10 @@ def __init__(self, deviceName, config):
         elif(dType == 'what'):
             deviceObj = self.load_display_driver(self.pkg_name, 'what')
             self._device = deviceObj.InkyWHAT(dColor)
+        elif(dType == 'auto'):
+            deviceObj = self.load_display_driver(self.pkg_name, 'auto')
+            self._device = deviceObj.auto()
+            self.mode = dColor = self._device.colour

```

Comment:
this will override any colour set in the config.. needs fixing.

Reply:
You could not set the `self.mode` variable here to keep the user defined color. The check further down in the code will compare the board's reported color with what the user has set (to avoid setting "yellow" on a "red" pHat). This would also keep the original bw setting as the default unless the user has set otherwise.

Reply:
Will do. 

BTW, why is black and white the default for all displays?